src/datastructure.h

```diff
@@ -148,6 +148,7 @@ bool is_blocked(const enum query_status status) __attribute__ ((const));
 bool is_cached(const enum query_status status) __attribute__ ((const));
 const char *get_blocked_statuslist(void) __attribute__ ((pure));
 const char *get_cached_statuslist(void) __attribute__ ((pure));
+const char * get_permitted_statuslist(void) __attribute__ ((pure));
```

Comment:
Is this correct?

Shouldn't be:
```suggestion
const char *get_permitted_statuslist(void) __attribute__ ((pure));
```